schemas/README.md

```diff
@@ -0,0 +1,51 @@
+# Maintenance
+
+## Introduction
+At some point in time there will be an increase an topics or even a change to the existing topic structure. If at any point one of these happen, this will require a change to the Avro schemas in further requiring a change in the structs that represent the Avro schemas. These structs are used by the kafka-management-tool in order to map the unmarshalled JSON message from with the specified topic.
```

Comment:
This doesn't look correct - "an increase an topics"

Reply:
ahh yes, meant to be `in`... good spot